source/fundamentals/query-data.txt

```diff
@@ -169,9 +169,105 @@ on the ``name`` field.
    When sorting on fields with a boolean value, entities with a field value of
    ``false`` show before those with a value of ``true``.
 
+Scalar Aggregations
+-------------------
+
+The {+provider-short+} supports the following scalar aggregation methods:
+
+- ``Count()``: Returns the number of elements in a collection
+- ``LongCount()``: Returns the number of elements in a collection as a ``long``
+- ``Any()``: Returns ``true`` if any elements in a collection match the search criteria
+- ``Max()``: Returns the maximum value of a specified field in a collection
+- ``Min()``: Returns the minimum value of a specified field in a collection
+- ``Sum()``: Returns the sum of a specified field in a collection
+- ``Average()``: Returns the average of a specified field in a collection
+
+The following sections show examples of each of the preceding methods.
+
+Count
+~~~~~
+
+The following example uses the ``Count()`` method to count the number of planets in the
+``Planets`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-examples/aggregation.cs
+   :language: csharp
+   :start-after: // start-count
+   :end-before: // end-count
```

Comment:
S: include output if its additive

Reply:
I'm not sure output would really add much value here since in these it would just be an arbitrary list of planet names or numbers